### ISSUE TITLE
Brazilian Portuguese Translation

### ISSUE THREAD

**Issue by localhostpedro** [clojure/clojure-site#234]
Hello Everyone

I´d like to contribute to Clojure , creating a Brazilian Portuguese version of the website, to help spreading the word of Clojure among Brazilian developers. 

Can anyone tell me if is possible , or point me the right direction ?

Best regards

Pedro

**Comment by iku000888** [comment]
@localhostpedro 

Hi there! 

There is this on going issue regarding hosted official translations https://github.com/clojure/clojure-site/issues/1

Apart from that there is an independent effort for Japanese translations https://github.com/japan-clojurians/clojure-site-ja which may be interesting  for other languages in terms of tools/project setup.

**Comment by localhostpedro** [comment]
Thank you @iku000888 . I'll keep looking at this issue. Maybe using Rails translation system would help to create a more consistent translation experience. I've been using for some time and its easy to keep everything up-to-date using the same translation structure.

**Comment by localhostpedro** [comment]
Actually I already started translated some content for personal studies and to organize the Clojure Bridge event in my city. Next step is to get some feedback from users: https://github.com/clojurebridgemaringa/Visao-Geral/wiki

But this information must be in the main site, make it more accessible.

**Comment by iku000888** [comment]
Nice.

I think the best thing that can happen now is to nudge @puredanger enough times to make the 'hosted on clojure.org' happen.

**Comment by puredanger** [comment]
Yes, I would like to make that happen, just a matter of finding a chunk of time. :)

I think if I can get the base repo and automation going, then you would be able to work on populating it independently.

**Comment by localhostpedro** [comment]
That would be amazing @puredanger  , thank you for the reply.Ping me anytime or add me as contributor.
